coderd/util/maps/maps.go

```diff
@@ -0,0 +1,14 @@
+package maps
+
+// Subset returns true if all the keys of a are present
+// in b and have the same values.
+func Subset[T, U comparable](a, b map[T]U) bool {
+	for ka, va := range a {
+		if vb, ok := b[ka]; !ok {
+			return false
+		} else if va != vb {
+			return false
+		}
+	}
+	return true
+}
```

Comment:
review: I couldn't find a stdlib function for this, so rolled my own. I'd love to get rid of this.